src/escalus_fresh.erl

```diff
@@ -2,7 +2,9 @@
 -export([story/3,
          story_with_client_list/3,
          story_with_config/3,
-         create_users/2]).
+         create_users/2,
+         given_fresh_specs/2,
```

Comment:
The `given_` prefix in the original function's name is inspired by [BDD](https://en.wikipedia.org/wiki/Behavior-driven_development), therefore makes sense in the test suite as it describes the assumption about the particular situation under test.
However, this prefix is not very meaningful in a library function. I think we can just rename these two functions to: `fresh_spec` and `fresh_specs`.

Reply:
I will call them freshen_spec/freshen_specs so that it both differs from get_fresh_spec (which returns the fresh spec while the former returns config with fresh specs) and is a verb.